keras_hub/src/models/backbone.py

```diff
@@ -188,11 +194,13 @@ def save_to_preset(self, preset_dir):
 
     def get_lora_target_names(self):
         """Returns list of layer names which are to be LoRA-fied.
-
-        Subclasses can override this method if the names of layers to be
-        LoRa-fied are different.
         """
-        return ["query_dense", "value_dense", "query", "value"]
+        return self._lora_target_names
+    
+    def set_lora_target_names(self, target_names):
```

Comment:
We should not have a setter like this. In fact `get_lora_target_names` is really only for model implementations and should probably be renamed `default_lora_layer_names` in this cl (do a find and replace to clean this up everywhere).